monkestation/code/datums/quirks/positive_quirks.dm

```diff
@@ -47,3 +47,28 @@
 /datum/quirk/gourmand/remove()
 	var/mob/living/carbon/human/holder = quirk_holder
 	holder.max_food_buffs --
+
+/datum/quirk/fluffy_tongue
+	name = "Fluffy Tongue"
+	desc = "After spending too much time watching anime you have developed a horrible speech impediment."
+	value = 5
+	icon = FA_ICON_CAT
+
+/datum/quirk/fluffy_tongue/add()
+	. = ..()
+	RegisterSignal(quirk_holder, COMSIG_MOB_SAY, PROC_REF(handle_speech))
```

Comment:
Signal is never unregistered on loss of quirk

Reply:
Ah, I knew I was missing something.